github/issues.go

```diff
@@ -131,6 +132,9 @@ func (s *IssuesService) listIssues(u string, opt *IssueListOptions) ([]Issue, *R
 		return nil, nil, err
 	}
 
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeReactionsPreview)
```

Comment:
If you read the GitHub API docs carefully, it sounds like a bug on their end. From https://developer.github.com/v3/issues/#list-issues:

> This endpoint may also return pull requests in the response. If an issue is a pull request, the object will include a pull_request key.

But with this Accept header set, `pull_request` key is included in all objects, both issues and PRs (but the contents of `pull_request` do vary, of course).